test_runner/fixtures/neon_fixtures.py

```diff
@@ -420,7 +447,7 @@ def __init__(
         self.default_branch_name = default_branch_name
         self.env: Optional[NeonEnv] = None
         self.keep_remote_storage_contents: bool = True
-        self.neon_binpath = neon_binpath
+        self.neon_binpath = neon_binpath.absolute()
```

Comment:
Probably should be done in the `neon_binpath` fixture

Reply:
it's not needed at all